.github/workflows/openapi-generator.yaml

```diff
@@ -140,8 +140,23 @@ jobs:
         with:
           name: openapi-generator-cli.jar
           path: modules/openapi-generator-cli/target
+      - name: Delete samples that are entirely generated
+        run: |
+          rm -rf samples/client/petstore/csharp/generichost/net8/AllOf
+          rm -rf samples/client/petstore/csharp/generichost/net8/AnyOf
+          rm -rf samples/client/petstore/csharp/generichost/net8/AnyOfNoCompare
+          rm -rf samples/client/petstore/csharp/generichost/net8/FormModels
+          rm -rf samples/client/petstore/csharp/generichost/net8/NullReferenceTypes
+          rm -rf samples/client/petstore/csharp/generichost/net8/OneOf
+          rm -rf samples/client/petstore/csharp/generichost/net8/Petstore
+          rm -rf samples/client/petstore/csharp/generichost/net8/SourceGeneration
+          rm -rf samples/client/petstore/csharp/generichost/net8/UseDateTimeForDate
+          rm -rf samples/client/petstore/csharp/generichost/standard2.0/Petstore
       - name: Generate samples
-        run: bash bin/generate-samples.sh
+        run: |
+          bash bin/generate-samples.sh
+          # when a sample is deleted, you have to generate it twice for all files to get created
+          bash bin/generate-samples.sh
```

Comment:
this PR includes lots of changes not belonging to your original PR

can you please submit a new one with just the change for just "allArgConstruct for java"?

Reply:
sorry. I screwed up the merges.

Is it better now?